### DISCUSSION TITLE
AS, AD, PS,PD, NRD, NRS parameters

### DISCUSSION THREAD

**Discussion by eskiyerli** [Xyce/Xyce#34] [Q&A]
In a multi-finger MOSFET, are these parameters per finger or total number for all fingers?

**Comment by Karlsefni2012** [comment]
Off the top of my head, I don't know. As I am sure you know we didn't develop the FinFET models, and they are from the BSIM group at Berkeley.  So this is something that should be in the BSIM-CMG documentation somewhere.  https://bsim.berkeley.edu/models/bsimcmg/

**Reply by eskiyerli** [reply]
OK. I should check the documents then. My question was not about specifically about FINFET models. I think a few other models are also using such parameters.
